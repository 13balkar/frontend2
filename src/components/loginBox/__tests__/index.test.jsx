import * as React from 'react';
import { render } from '@testing-library/react';
import LoginBox from '../';
jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('LoginBox', () => {
  it('should render the loginBox', () => {
    const { asFragment } = render(<LoginBox />);
    expect(asFragment()).toMatchSnapshot();
  });
});
