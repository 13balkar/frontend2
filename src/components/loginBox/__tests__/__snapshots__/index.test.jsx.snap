// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBox should render the loginBox 1`] = `
<DocumentFragment>
  <div
    class="login-box"
  >
    <h1>
      Login to your CMS+ account
    </h1>
    <div
      class="card"
    >
      <label>
        Email
      </label>
      <input
        type="text"
        value=""
      />
      <label>
        Password
      </label>
      <input
        type="password"
        value=""
      />
      <button>
        Login
      </button>
      <div
        class="links"
      >
        <p>
          Forgot password?
        </p>
        <p>
          Create an account
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
